modules/ROOT/pages/introduction/intro-connector-configuration-overview.adoc

```diff
@@ -0,0 +1,148 @@
+
+To use a global element, create one global element to configure your connection, configuration, or transport details for the connector. Reference the global element from other instances of the connector in the flow to ensure consistency across flow elements.
+
+In addition to the benefits of reusability and consistency, global elements also make connector maintenance easier. If you must change a connector's configuration, you can change the configuration once in the global element and it will apply to all connectors that reference the global element.
+
+Most connectors support the use of a global element for authentication and other functions. For information about configuring a global element for a specific connector, see the user guide for that connector.
+
+[[property-placeholders]]
+=== Use Property Placeholders for Property Values
+
+Instead of using static values for a connector configuration in Studio, you can use http://ant.apache.org[Ant-style] property placeholders, as shown in this Studio example:
+
+.Use ANT-style property placeholders for maintainability
+image::introduction/intro-config-overview-properties.png[Configuring property placeholders]
+
+* The first item is a placeholder for the Amazon SQS access key.
+* The second item is a placeholder for the Amazon SQS secret key.
+
+Using property placeholders improves the maintainability of your applications because they enable you to change configuration values in one place. If you don't use property placeholders and configuration parameter values change, you must search for changed parameters in the connector configuration screens or configuration XML.
+
+To make the values for the placeholders available, create a YAML file (`.yaml`) or properties (`.properties`) file to contain the values, place the file in `src/main/resources` in the Mule project, and then reference the file from the connector configuration.
+
+To reference a file that stores property values in Studio:
+
+. In the connector global element *Choose Global Type* screen, search for the element called *Configuration properties*.
+. Select *Configuration properties*.
+. Click *OK*.
+. Click *…​* to the right of the *File* field and navigate to your YAML or properties file.
+
+To reference a file that stores property values when configuring a connector using XML:
+
+. Include a `<configuration-properties>` element inside the `<mule>` element.
+. Set the `file` parameter with your properties file name. For example:
+
+[source,xml,linenums]
+----
+<mule>
+  <configuration-properties
+    file="myConfiguration.yaml"
+    doc:name="Configuration properties"
+    />`
+  ...
+</mule>
+----
+
+For more information about using property placeholders, see xref:mule-runtime::create-and-configure-your-properties-file.adoc[Configuring Property Placeholders].
+
+[[test-connectivity]]
+=== Test the Connectivity of the Connector
```

Comment:
Is this part of the process in the order that a user would perform it, relative to configuring the connector?

Reply:
I wonder if this task belongs in this topic set. 
Seems like it should be elsewhere, and that we should mention as a requirement for configuration that the connector is working....

Reply:
Ah...nevermind. The config UI includes a testing control.